flake.nix

```diff
@@ -71,6 +70,8 @@
               src = craneLib.cleanCargoSource (craneLib.path ./.);
               doCheck = false;
 
+              cargoExtraArgs = "-p trycp_server";
```

Comment:
Select only what we want to build. It was actually building wasms too and creating a larger bundle than necessary